src/main/java/teammates/common/util/Templates.java

```diff
@@ -196,4 +196,54 @@ public static String populateTemplate(String template, String... values) {
                 FileHelper.readResourceFile("feedbackSessionTeamEvaluationTemplate.json");
     }
     
+    public static class FeedbackQuestionDetailsConsts {
```

Comment:
Sorry I didn't get you. Do I need to change class names from these ? If yes, I think FeedbackQuestionDetailsSlots is better.